### PR TITLE
Refactor to use laravel-webhook-client under the hood

### PR DESCRIPTION
**Pull request by riasvdv**

### CODE REVIEW

File: src/StripeWebhooksServiceProvider.php [hunk 1]
```diff
@@ -15,16 +15,8 @@ public function boot()
...
-            $timestamp = date('Y_m_d_His', time());
-
-            $this->publishes([
-                __DIR__.'/../database/migrations/create_stripe_webhook_calls_table.php.stub' => database_path('migrations/'.$timestamp.'_create_stripe_webhook_calls_table.php'),
-            ], 'migrations');
-        }
-
         Route::macro('stripeWebhooks', function ($url) {
-            return Route::post($url, '\Spatie\StripeWebhooks\StripeWebhooksController');
+            return Route::post($url, StripeWebhooksController::class);
```
Comment (freekmurze) [inline src/StripeWebhooksServiceProvider.php]: I'm thinking this needs to be a string starting with `\` so also works with in apps where a namespace is set up in the service provider.

**Comment by riasvdv** [comment]
Tested locally with `valet share` and a Stripe test account

- Readme instructions all work
- Webhook calls are correctly saved to the `webhook_calls` table
- Stripe job is processed
- Mapping of event --> job works correctly
- Payload is saved
- Correct errors are returned if something is wrong with signing

Looks like everything works 👍

**Comment by freekmurze** [comment]
Looks great! I'll go over it one more time before tagging 2.0 👍

**Comment by freekmurze** [comment]
Thanks, great job @riasvdv

**Comment by amadeann** [comment]
This refactor removed the possibility to customize the `WebhookCall` model via `config('stripe-webhooks.model')` (and consequently the `webhook_calls` table name).

**Comment by riasvdv** [comment]
@amadeann Thanks for letting us know, I've fixed it in the latest release

**Pull request merged**